modules/costs/app/components/my/time_tracking/time_entries_list_component.rb

```diff
@@ -85,6 +88,10 @@ def skip_column?(column)
           false
         end
       end
+
+      def can_create_time_entry?
+        User.current.allowed_in_any_work_package?(:log_own_time) || User.current.allowed_in_project?(:log_time)
```

Comment:
I got an exception on this line."wrong number of arguments (given 1, expected 2)"

when calling `allowed_in_project?(permission, projects_to_check)`

I have the feeling that there are not a lot of unit tests to check for these cases.

Reply:
Oh ... Needs to be `in_any_project` 👍